README.md

```diff
@@ -5,40 +5,44 @@ Example Jupyter notebook source files for the BioData Catalyst consortium. These
 Jupyter notebook `.ipynb` files are generated from source files with the
 [callysto](https://github.com/xbrianh/callysto) library. 
 
-## Source File Schema
-The first line of each notebook source file contains the publish directive, a string with the following format:
-```
-# publish to "{my workspace name}" "{my notebook name}"
-```
-where `{my workspace name}` should be replaced with the name of the workspace, and `{my notebook name}` should be
-replaced with the name of the notebook.
+## Notebook Source Schema
+A notebook is defined by creating a directory under ${REPO_ROOT}/noteboks containing
+  - `main.py`, the source script for the notebook in [callysto](https://github.com/xbrianh/callysto) format.
+  - `requirements.txt`, pip-installable requirements needed for notebook execution.
+  - `publish.txt`, a list of destination Google Storage URLs where the notebook will be published.
 
-Notebook source files are executable Python scripts. The contents of cells are denoted with the callysto Python
-context managers:
+### The Callysto Format
+Notebook source files, `main.py`, are executable Python scripts. Contents of cells are denoted with the callysto
+Python context managers:
 ```
 with callysto.Cell("{mode}"):
 	...
 ```
 where `{mode}` is `python` or `markdown`. Python statements outside of callysto contexts will not be rendered into
-cells. Test code, or code for preparing mock environments, should not be placed in callysto contexts.
+cells. Test code or mock environments should not be placed outside callysto contexts.
```

Comment:
`outside of`

Reply:
>Outside of tends to be more commonly used in the US than in Britain, where outside usually suffices, but, like its cousin off of, it is colloquial and not recommended for formal writing

https://english.stackexchange.com/questions/9700/outside-or-outside-of